cost/scheduled_report_markUpsDowns/CHANGELOG.md

```diff
@@ -0,0 +1,2 @@
+v0.0
```

Comment:
Add the CHANGELOG header

Under the version v0.0, add **Not included in the Catalog**